.github/workflows/event-push-to-integ.yml

```diff
@@ -11,5 +11,5 @@ on:
 
 jobs:
   benchmark:
-    uses: ./.github/workflows/trigger-benchmark.yml
+    uses: ./.github/workflows/benchmark-runner.yml
```

Comment:
Following @K-Jo comment, do you think its better adding it to the merge queue flow rather than adding it on merge to integ?

Reply:
Not yet. We should first have the infrastructure to fail on regression. For now, let's stick to the way we always did it.